app/locales/nl.json

```diff
@@ -0,0 +1,102 @@
+    "maps_import_button_text": "Importeer eerdere locaties",
+    "maps_import_disclaimer": "COVID Safe Paths is niet aangesloten bij Google en zal nooit uw data delen.",
+    "maps_import_text": "Om te zien of u in contact bent geweest met een COVID-19 besmet persoon, voordat u deze applicatie gebruikte, importeer uw locatie geschiedenis.",
+    "maps_import_title": "Google Maps",
+    "nCoV2019_url_info": "Meer informatie over deze gegevens",
+    "news_subtitle": "Lees meer over de laatste COVID-19 nieuws op de website vaRead about the latest COVID updates from your health authority and in general.",
+    "news_title": "Laatste nieuws",
+    "no_data": "Geen gegevens",
+    "notification_2_weeks_ago": "2 weken geleden",
+    "notification_data_not_available": "Geen blootstelling gegevens beschikbaar.",
+    "notification_random_data_button": "Kies een gezondheidsinstantie",
+    "notification_title": "Twee wekelijks blootstellingsprofiel",
+    "notification_today": "vandaag",
+    "notification_warning_text": "Als er in uw regio een gezondheidsautoriteit actief is, kunt u zich abonneren om regelmatig updates van blootstellingsrisico's te ontvangen.",
+    "notifications_exposure_format": "{{daysAgo}} dagen geleden, kwam u voor {{exposureTime}} minuten in contact geweest met een besmet persoon.",
+    "notifications_exposure_format_today": "Vandaag bent u voor {{exposureTime}} minuten in contact geweest met een besmet persoon",
+    "notifications_exposure_format_yesterday": "Gister bent u voor {{exposureTime}} minuten in contact geweest met een besmet persoon",
+    "notifications_no_exposure": "Geen bekende blootstelling aan COVID-19 gedurende de laatste twee weken.",
+    "overlap_found_button_label": "Publieke gegevens geladen",
+    "overlap_no_results_button_label": "Publieke gegevens geladen",
+    "overlap_para_1": "Het groene pad is uw locatiegeschiedenis. De licht paarse cirkel staan voor publieke gegevens.",
+    "overlap_title": "Bekijk overlap",
+    "push_at_risk_message": "U bent in de buurt geeest met iemand met de diagnose COVID-19.",
+    "push_at_risk_title": "U loopt mogelijk risico",
+    "settings_title": "Dashboard",
+    "share_location_data": "Deel uw locatie gegevens",
+    "show_overlap": "Bekijk publieke gegevens",
+    "team": "Team",
+    "team_para": "Ons team bestaat uit een consortium van epidemiologen, ingenieurs, data wetenschappers, digitale privacy-evangelisten, professoren en onderzoekers van gerenommeerde instellingen, waaronder: MIT, Harvard, The Mayo Clinic, TripleBlind, EyeNetra, Ernst & Young en Link Ventures.",
+    "terms_of_use": "Gebruiksvoorwaarden",
+    "terms_of_use_url": "https://docs.google.com/document/d/1mtdal_pywsKZVMXLHjjj5eKznipPLP8sM1HwFTIhjo0/edit#",
+    "test_positive_subtitle": "Uw privégegevens kunnen worden overgedragen aan gezondheidsautoriteiten, een back-up worden gemaakt of anderszins worden gedeeld.",
+    "tested_positive_title": "Deel locatie geschiedenis"
+  }
```

Comment:
there are more new keys @danoszz will you have a look at the current en.json and make sure you have them?

Reply:
Yes, got it!